test/e2e/common/kubelet_etc_hosts.go

```diff
@@ -59,6 +62,10 @@ var _ = framework.KubeDescribe("KubeletManagedEtcHosts", func() {
 
 		By("Running the test")
 		config.verifyEtcHosts()
+
+		if config.tmpfile != nil {
```

Comment:
make a `cleanup()` method and do this in there as a step, just like setup()?

Reply:
done